app.js

```diff
@@ -1,18 +1,74 @@
-require('dotenv').config();
+require("dotenv").config();
 
-const express = require('express');
-const hbs = require('hbs');
+const express = require("express");
+const hbs = require("hbs");
 
 // require spotify-web-api-node package here:
+const SpotifyWebApi = require("spotify-web-api-node");
+console.log(process.env.CLIENT_ID);
```

Comment:
Its cool try to know if the things are working, but every time we create a Pull Request we should remove the console.logs that are not necessary, you can leave the ones that are inside catch